pandas_gbq/gbq_connector.py

```diff
@@ -11,6 +11,8 @@
 
 import numpy as np
 
+from . import environment
```

Comment:
Nit: We try to avoid relative imports.

```suggestion
from pandas_gbq import environment
```

Reply:
Updated, thanks!